provider/maas/environ.go

```diff
@@ -948,6 +951,21 @@ func (environ *maasEnviron) StartInstance(args environs.StartInstanceParams) (
 			return nil, errors.Annotatef(err, "cannot run instances")
 		}
 	}
+	go func() {
+		for {
+			instStatus := inst.Status()
+			if args.StatusCallback == nil {
+				logger.Errorf("no status callback to report %s status: %v", inst.Id(), instStatus)
+				<-clock.WallClock.After(statusPollInterval)
+				continue
```

Comment:
Well I thought that the log message would be of value for debugging purposes, I think I might make it a Debug level instead.